catalog/src/index.ts

```diff
@@ -0,0 +1,200 @@
+import type { Api } from "@codemod.com/workflow";
+import * as semver from "semver";
+
+type PackagesVersions = Record<
+	string,
+	{
+		usages: number;
+		versions: string[];
+	}
+>;
+
+type PackageJson = {
+	packageManager?: string;
+	dependencies?: Record<string, string>;
+	devDependencies?: Record<string, string>;
+};
+
+const isAlias = (version: string) => {
+	return version.match(/^p?npm:/)?.length === 1;
```

Comment:
alias can only start with `npm:`
```suggestion
	return version.startsWith('npm:');
```